nuget.config

```diff
@@ -1,7 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <clear />
```

Comment:
We can't remove this - this is a security thing (so you're always sure where you get packages from when building the project)

I'm not sure why this would be necessary to remove in the first place though - the only two places we should be pulling packages from are the two right below so that should be all that's needed for building the projects in this repo.

Reply:
WorkerExtensions.csproj is not part of the repo and hence the issue. If I remove that, it will not be able to see the new source added and will fail with the same error.
@lucyzhang929 Can you test and confirm this since am still not able to repro the issue and figuring it out?

Reply:
Resolved this offline through teams. Thanks again @Charles-Gagnon for explaining how nuget commands work.